util/feed.go

```diff
@@ -91,6 +91,9 @@ type Feed struct {
 // all Subscribers to the feed. Events published to a closed feed, or to a feed
 // with no Subscribers, will be ignored.
 func (f *Feed) Publish(event interface{}) {
+	if f == nil {
```

Comment:
Not sure I understand, using a `nil` feed would previously panic. Now it behaves closer to a closed feed.
This edit isn't part of some grand plan yet, but it would be convenient if a nil feed behaved like a dummy feed that doesn't do anything (but you can still subscribe to it). Otherwise various places need to do some dancing around this exceptional case. StoreEventFeed checks this all the time, for instance.
How do you feel about that?